test/jdk/java/awt/Focus/8073453/AWTFocusTransitionTest.java

```diff
@@ -50,6 +71,7 @@ public static void main(String[] args) throws Exception {
 
             robot.waitForIdle();
 
+            blockTillDisplayed(textField);
```

Comment:
Are you sure that this block is needed? How it is possible that the component which was made visible above in the createAndShowGUI() may be invisible here?